packages/core/admin/admin/src/content-manager/pages/ListView/components/Table.tsx

```diff
@@ -54,118 +58,119 @@ const CheckboxDataCell = ({ rowId, index }: CheckboxDataCellProps) => {
  * EntityActionsDataCell
  * -----------------------------------------------------------------------------------------------*/
 
-interface EntityActionsDataCellProps {
-  rowId: Entity.ID;
-  index: number;
-  canCreate?: boolean;
-  canDelete?: boolean;
-  setIsConfirmDeleteRowOpen: (isOpen: boolean) => void;
-  handleCloneClick: (id: Entity.ID) => () => void;
-}
-
-const EntityActionsDataCell = ({
-  rowId,
-  index,
-  canCreate = false,
-  canDelete = false,
-  setIsConfirmDeleteRowOpen,
-  handleCloneClick,
-}: EntityActionsDataCellProps) => {
-  const location = useLocation();
-  const { formatMessage } = useIntl();
-  const { trackUsage } = useTracking();
-  const { setSelectedEntries } = useTableContext();
-  const [{ query }] = useQueryParams<{ plugins?: Record<string, unknown> }>();
-
-  const itemLineText = formatMessage(
-    {
-      id: 'content-manager.components.ListViewTable.row-line',
-      defaultMessage: 'item line {number}',
-    },
-    { number: index + 1 }
-  );
-
-  return (
-    <Flex gap={1} justifyContent="end" onClick={stopPropagation}>
-      <IconButton
-        forwardedAs={Link}
-        onClick={() => {
-          trackUsage('willEditEntryFromButton');
-        }}
-        // @ts-expect-error – DS does not correctly infer props from the as prop.
-        to={{
-          pathname: rowId.toString(),
-          state: { from: location.pathname },
-          search: query.plugins ? stringify({ plugins: query.plugins }) : '',
-        }}
-        label={formatMessage(
-          { id: 'app.component.table.edit', defaultMessage: 'Edit {target}' },
-          { target: itemLineText }
-        )}
-        borderWidth={0}
-      >
-        <Pencil />
-      </IconButton>
-
-      {canCreate && (
-        <IconButton
-          onClick={handleCloneClick(rowId)}
-          label={formatMessage(
-            {
-              id: 'app.component.table.duplicate',
-              defaultMessage: 'Duplicate {target}',
-            },
-            { target: itemLineText }
-          )}
-          borderWidth={0}
-        >
-          <Duplicate />
-        </IconButton>
-      )}
-
-      {canDelete && (
-        <IconButton
-          onClick={() => {
-            trackUsage('willDeleteEntryFromList');
-            setSelectedEntries([rowId]);
-            setIsConfirmDeleteRowOpen(true);
-          }}
-          label={formatMessage(
-            { id: 'global.delete-target', defaultMessage: 'Delete {target}' },
-            { target: itemLineText }
-          )}
-          borderWidth={0}
-        >
-          <Trash />
-        </IconButton>
-      )}
-    </Flex>
-  );
-};
+// interface EntityActionsDataCellProps {
+//   rowId: Entity.ID;
```

Comment:
Todo?

Reply:
It'll be re-added in this feature branch we've made :)